lib/LaTeXML/resources/XSLT/LaTeXML-picture-xhtml.xsl

```diff
@@ -95,8 +95,17 @@
       <xsl:call-template name="add_classes"/>
       <xsl:call-template name="copy_foreign_attributes"/>
       <xsl:apply-templates select="." mode="add_RDFa"/>
+      <xsl:if test="@imagedepth | svg:svg/@style">
+        <xsl:attribute name="style">
+          <xsl:if test="@imagedepth">
+            <xsl:value-of select="concat('vertical-align:',-@imagedepth,'px')"/>
+            <xsl:if test="svg:svg/@style">;</xsl:if>
+          </xsl:if>
+          <xsl:value-of select="svg:svg/@style"/>
+        </xsl:attribute>
+      </xsl:if>
       <!-- but copy other svg:svg attributes -->
```

Comment:
It's probably my fault for being slow with the PR reviews, but now we've got different variants of adjusting depth or baseline in the XSLT (#2397)
If we had a `@depth`, and used `<xsl:call-templates name="add_attributes"/>`, instead of `add_classes`, we'd get the adjustment for free, I think. Do we get too many other CSS things added?

Reply:
Yes, unfortunately they overlap, but they are trying to do the same thing. I wager that this version here is the better one, for instance because `-` is outside of the string!

Reply:
> we'd get the adjustment for free

I didn't reply to this one, but of course that sounds like a good idea (adding `@depth` to the schema and implementing the special behaviour in `add_attributes`).